src/selectors/entities/channels.js

```diff
@@ -222,35 +222,37 @@ export const getOtherChannels = createSelector(
 export const getChannelsByCategory = createSelector(
     getCurrentChannelId,
     getMyChannels,
+    getConfig,
```

Comment:
perhaps think of passing just the property from the config that you need, that way if something else in the config changes the selector might not need to re-calculate